pos_sale_order/res_partner_data.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<openerp>
+    <data noupdate="0">
+
+        <record model="res.partner" id="res_partner_anonymous">
```

Comment:
Hi, 
I'm not in favor of partner created by xml ids. This kind of design will not support multicompany context, or will not be flexible in a multishop context. what about to make this feature more customizable : 
* add a m2o required field in pos.config, named 'anonymous_partner_id' ; 
* get this partner, instead of getting this current partner by xmlid_to_res_id; 

**data >> demo data**
* set this xml file in 'demo' part of your file openerp ;
* in another demo file, set :

        <record model="pos.config" id="point_of_sale.pos_config_main" >
            <field name="anonymous_partner_id" ref="res_partner_anonymous" />
        </record>